homeassistant/components/velbus/config_flow.py

```diff
@@ -0,0 +1,94 @@
+"""Config flow for the Velbus platform."""
+import velbus
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PORT, CONF_NAME
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.util import slugify
+
+from .const import DOMAIN
+
+
+@callback
+def velbus_entries(hass: HomeAssistant):
+    """Return connections for Velbus domain."""
+    return set((entry.data[CONF_PORT]) for
+               entry in hass.config_entries.async_entries(DOMAIN))
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class VelbusConfigFlow(config_entries.ConfigFlow):
+    """Handle a config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+
+    def __init__(self) -> None:
+        """ Initialize the velbus config flow."""
+        self._errors = {}
+
+    def _create_device(self, name: str, prt: str):
+        """ The call to create the device itself."""
+        return self.async_create_entry(
+            title=name,
+            data={
+                CONF_PORT: prt
+            }
+        )
+
+    def _test_connection(self, prt):
+        """ This call will load the velbus controller with the port"""
+        try:
+            controller = velbus.Controller(prt)
+        except Exception:  # pylint: disable=broad-except
+            self._errors[CONF_PORT] = 'connection_failed'
+            return False
+        else:
```

Comment:
We don't need `else:` since we return in the exception block. We can outdent below.